### PR TITLE
adding photo & about Ina

### PR DESCRIPTION
**Pull request by ina-stoyanova**
Hey, if you wouldn't mind having a look of how it sounds/looks? Much appreciated :)

### CODE REVIEW

**Review by brikis98 — APPROVED** [review]
LGTM!

@josh-padnick @eak12913 @oredavids Could one of you merge & release? Thx!

**Comment by josh-padnick** [comment]
LGTM, but @ina-stoyanova  did you want to add your title as well?

**Comment by ina-stoyanova** [comment]
Ah good spot, Josh! Thanks - added it now :)

**Comment by josh-padnick** [comment]
These GitHub Pull Requests actually include a direct link to a preview environment. See the very last entry:

image

I checked that out and still didn't see the title, then saw the issue was a typo. Once you fix that, I think we're all set!

File: _data/team.yml [hunk 1]
```diff
@@ -174,6 +174,18 @@
     also a hiker, boulderer, and board game enthusiast who dabbles in game
     design himself.
 
+- name: Ina Stoyanova
+  url: "https://www.linkedin.com/in/ina-stoyanova/"
+  titile: Software Engineer
```
Comment (josh-padnick) [inline _data/team.yml:179]:
```suggestion
  title: Software Engineer
```

**Review by eak12913 — APPROVED** [review]
LGTM! I didn't see the typo in `title` yesterday and thought that something was broken with our preview deploy. So I started looking into that...couldn't find anything wrong. Then saw the commit this morning and was like: "OOOoooooooo!" Glad it was so simple!

**Comment by ina-stoyanova** [comment]
Hahaha, yes @eak12913 - I missed it too when I pushed the change on Friday - just checked locally if it looked ok. :) Also glad it's a typo on my end! 

Next steps: I can now merge it, but won't be able to deploy it yet as I don't have access to AWS for the website. Much appreciated if anyone could help me out with this :)

**Comment by eak12913** [comment]
Hey @ina-stoyanova, feel free to merge this PR and then head over to the releases page: https://github.com/gruntwork-io/gruntwork-io.github.io/releases

In order to release the site to production all you have to do is to create a new release on that releases page. The CI/CD process will then take care of pushing that to production.

**Comment by eak12913** [comment]
@ina-stoyanova - Deploy looks good!

image

**Comment by ina-stoyanova** [comment]
Thanks @eak12913

**Pull request merged**